projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-update-media-v1-1-endpoint.php

```diff
@@ -125,10 +131,45 @@ function callback( $path = '', $blog_id = 0, $media_id = 0 ) {
 			}
 		}
 
+		$result = $this->handle_video_meta( $media_id, $input, $item );
+
+		if ( is_wp_error( $result ) ) {
+			return $result;
+		}
+
 		$insert['ID'] = $media_id;
 		wp_update_post( (object) $insert );
 
 		$item = $this->get_media_item_v1_1( $media_id );
 		return $item;
 	}
+
+	/**
+	 * Persist the videopress metadata.
+	 *
+	 * @param string   $media_id The ID of the video.
+	 * @param array    $input    The request input.
+	 * @param stdClass $item     The response item.
+	 *
+	 * @return bool|WP_Error
+	 */
+	public function handle_video_meta( $media_id, $input, $item ) {
+		if ( ! class_exists( \Videopress_Attachment_Metadata::class ) ) {
+			return false;
+		}
+
+		if ( ! Videopress_Attachment_Metadata::is_videopress_media( $item ) ) {
```

Comment:
Nitpick: Should we use the root `\` namespace here as well in order to be consistent with the rest of the code?
```suggestion
		if ( ! \Videopress_Attachment_Metadata::is_videopress_media( $item ) ) {
```